pkg/controller/dnsendpoint/dnsendpoint_controller.go

```diff
@@ -211,7 +210,7 @@ func (r *ReconcileDNSEndpoint) Reconcile(ctx context.Context, request reconcile.
 	}
 
 	if rootDomain == "" {
-		dnsLog.WithField("domain", fullDomain).Error("no scraper for domain found, skipping reconcile")
+		dnsLog.Error("no scraper for domain found, skipping reconcile")
```

Comment:
[Unrelated, latent.] The domain field is already in the logger -- see L197 above.